src/current/v24.3/data-resilience.md

```diff
@@ -0,0 +1,110 @@
+- [**Fault tolerance**]({% link {{ page.version.version }}/demo-fault-tolerance-and-recovery.md %}): Capabilities built in to CockroachDB to perform routine maintenance operations without any affect on performance. For example, [online schema changes]({% link {{ page.version.version }}/online-schema-changes.md %}), [WAL failover]({% link {{ page.version.version }}/cockroach-start.md %}#write-ahead-log-wal-failover).
+- **Logical data replication (LDR)** (Preview): A cross-cluster replication tool between multiple active CockroachDB clusters, which supports a diverse range of topologies. LDR provides eventually consistent, table-level replication.
+
+### Choose a high availability strategy
+
+CockroachDB uses synchronous, built-in Raft replication to create and distribute copies of data, ensuring consistency across the data copies. The database can tolerate nodes going offline without service interruption, whether in a single-region or [multi-region]({% link {{ page.version.version }}/multiregion-overview.md %}) cluster.
+
+<table class="comparison-chart">
+  <tr>
+    <th></th>
+    <th>Single-region replication (synchronous)</th>
+    <th>Multi-region replication (synchronous)</th>
+    <th>Logical data replication (asynchronous)</th>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>RPO</b>
+    </td>
+    <td>0 seconds</td>
+    <td>0 seconds</td>
+    <td>Immediate mode 0.5 seconds<br>Transactional mode ~10s of seconds</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>RTO</b>
+    </td>
+    <td>Zero RTO<br>Potential increased latency for 1-9 seconds</td>
+    <td>Zero RTO<br>Potential increased latency for 1-9 seconds</td>
+    <td>Zero RTO<br>Add application cutover time</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Write latency</b>
+    </td>
+    <td>Region-local write latency<br>p50 latency < 5ms (for multiple availability zones in us-east1)</td>
+    <td>Cross-region write latency<br>p50 latency > 50ms  (for a multi-region cluster in us-east1, us-east-2, us-west-1)</td>
+    <td>Region-local latency depending on design<br>p50 latency < 5ms (for multiple availability zones in us-east1)</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Recovery</b>
+    </td>
+    <td>Automatic</td>
+    <td>Automatic</td>
+    <td>Semi-automatic</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Hardware cost</b>
+    </td>
+    <td>1x<br>Networking between availability zones</td>
+    <td>1x<br>Networking between datacenters</td>
+    <td>2x<br>Networking between availability zones or datacenters</td>
+  </tr>
+
+  <tr>
+    <td class="comparison-chart__feature">
+      <b>Minimum regions</b>
+    </td>
+    <td>1</td>
+    <td>3</td>
+    <td>2</td>
```

Comment:
Do we really require two regions? Can we do this with two zones?

Reply:
I'll defer to @alicia-l2 or @awoods187 on this since this material comes from their presentations.

Reply:
You need two clusters but they could be in the same zone.

Reply:
From a resiliency perspective I think its more useful to talk about this from a region POV. Its possible to even run LDR within the same cluster but most people are looking to use LDR to survive a region outage, where they have two single region clusters replicating to each other.

Reply:
actually thinking about it more, what if we swap the "fault tolerance" and "minimum regions" rows, and then say "minimum regions to achieve fault tolerance?" so its more clear that with 2 and 3 regions, you can survive a region outage?

Reply:
Done, thanks Alicia